internal/cmd/io_binary_test.go

```diff
@@ -70,6 +66,45 @@ func Test_newBinaryCmd_Key_Value_Pair(t *testing.T) {
 	}
 }
 
+func Test_newBinaryCmd_Unsafe_Path(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	type config struct {
+		name          string
+		urlString     string
+		expectedError error
+	}
+
+	for _, cfg := range []*config{
+		{
+			name:          "UNC_Path_With_Back_Slashes",
+			urlString:     `binary:///\server\share\executable`,
```

Comment:
Should this path have 2 backslashes as in `binary://\\server\share\executable`?

Reply:
Hmm , this should work too since sanitizePath() calls filepath.Clean() which should parse "/\server\share\executable" to "\\server\share\executable" which hits the new if condition @anmaxvl  added to check if prefix "\\" exists. 
Am I missing smth?